CHANGELOG.md

```diff
@@ -128,4 +128,3 @@ Packages with other changes:
  - **FIX**: use app documents directory to store the user token if possible (#88).
  - **FIX**: Restore pub points (#75).
  - **FEAT**: Insights (#62).
-
```

Comment:
Restore the trailing line here to prevent this file from showing up in the diff.